src/main/target/common_fc_pre.h

```diff
@@ -76,6 +76,12 @@
 #define DEFAULT_AUX_CHANNEL_COUNT       6
 #endif
 
+#ifdef USE_ITCM_RAM
+#define CRITICAL_SECTION __attribute__((section(".critical_code")))
```

Comment:
Agreed. My first thought when hearing the term Critical Section is the definition @ledvinap points to.

While I generally have a preference for logical names rather than physical names, I can't think of a good logical name. So I suggest we use the name `ITCM_SECTION` - at least that has the benefit that it is clear to a reader what is going on.